### PR TITLE
Add possibility to set autofocus to input after mounting

### LINKED ISSUE #1044: Add possibility to set autofocus to input after mounting

### PR DESCRIPTION
**Pull request by DenisVorop**
## PR includes

- [x] Feature

## Related issues

Resolve #1044

### CODE REVIEW

File: src/harmony/Input/Input.tsx [hunk 1]
```diff
@@ -49,10 +60,25 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(
...
+
+        useEffect(() => {
+            if (autoFocus && inputRef.current) {
+                inputRef.current.focus();
+                setFocused(true);
+            }
+            return () => {
+                setFocused(false);
+            };
+        }, [autoFocus, focused]);
```
Comment (tsumo) [inline src/harmony/Input/Input.tsx:80]: Effect depends on focused, setFocused is called inside effect. Looks like a cycle

Comment (DenisVorop) [reply src/harmony/Input/Input.tsx:80]: Yes, but the condition is preventing it from happening

Comment (tsumo) [reply src/harmony/Input/Input.tsx:80]: Even with condition it's going to fire twice on the first render.
And what is the reason for saving focused state in the hook?

Comment (DenisVorop) [reply src/harmony/Input/Input.tsx:80]: If the component has been mounted, the focus is not assigned when it is reopened

**Review by tsumo — APPROVED** [review]

**Review by Katrin-kudryash — APPROVED** [review]

**Pull request merged**